fetch_runs.py

```diff
@@ -45,6 +52,13 @@
     with open(phone_uuid_path, "r") as f:
         phone_uuids = PhoneNumberUuidTable.load(f)
 
+    # Load test contacts path if set, otherwise default to empty set.
```

Comment:
I have a preference for a slightly different coding style here:

```
    test_contacts = set()
    if test_contacts_path is not None:
        with open(test_contacts_path, "r") as f:
            test_contacts.update(json.load(f))
```
This pulls the identity out of conditional which make the code easier to read through